backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3916,5 +3916,23 @@ databaseChangeLog:
             replaceIfExists: true
             fullDefinition: false
             selectQuery: SELECT id, event_timestamp, request_id, api_user_id, is_admin_user, user_permissions, organization_id, patient_link_id, response_code FROM ${database.defaultSchemaName}.api_audit_event
-        - sql: |
-            GRANT SELECT ON TABLE ${database.defaultSchemaName}.api_audit_event_no_phi TO ${noPhiUsername};
\ No newline at end of file
+        - sql: 
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.api_audit_event_no_phi TO ${noPhiUsername};
+  - changeSet:
+      id: update-no-phi-user-view-of-org-queue-table
+      author: bob@skylight.digital
+      comment: Add the organization_queue's is_deleted column to the view permissions of the no-phi-user so we can see it in metabase
+      changes:
+         - tagDatabase:
+            tag: update-org-queue-view
```

Comment:
We try to keep the id and tag identical for easier rollbacks, could you change the tag to `update-no-phi-user-view-of-org-queue-table`?